### PR TITLE
[2.19.x] G-9380 Unable to Edit a Buffer for MGRS Point Radius Search

### PR DESCRIPTION
**Pull request by abel-connexta**
ddf-ui PRs:
3.4.x codice/ddf-ui#447

---------------------
#### What does this PR do?
The user cannot manually enter numbers into the buffer field for MGRS Point Radius. User also cannot edit numbers that have been input into the MGRS Point Radius buffer that have been entered by drawing the geometry on the map.

#### Who is reviewing it? 

@andrewzimmer 
@hayleynorton
@cassandrabailey293 
@zta6 
@Lambeaux 
#### How should this be tested?

• build / run / profile install / upload sample data

Method 1
1. From any of these geometry field types (anyGeo, location, center), select Point Radius.
2. Select the MGRS coordinate format
3. Place cursor in Radius value box and enter a number from buffer radius
4. Ensure no error occurs and able to enter radius number

Method 2
1. From any of these geometry field types (anyGeo, location, center), select Point Radius.
2. Select the MGRS coordinate format
3. Click the Draw button and draw a Point Radius on the map. This will input numbers into the Radius buffer field.
4. Try to edit those numbers
5. Ensure no error occurs and able to edit radius number
#### What are the relevant tickets?
Fixes: #____

##### Before
Before-G-9380

##### After
After-G-9380

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Review by mojogitoverhere — APPROVED** [review]

File: ui/packages/catalog-ui-search/src/main/webapp/react-component/location/point-radius.js [hunk 1]
```diff
@@ -237,9 +237,16 @@ const PointRadiusUsngMgrs = props => {
           type="number"
           value={String(radius)}
           onChange={value => {
-            setRadiusError(validateGeo('radius', value))
             setState({ ['radius']: value })
           }}
+          onBlur={e =>
```
Comment (cassandrabailey293) [inline ui/packages/catalog-ui-search/src/main/webapp/react-component/location/point-radius.js:242]: huh, it looks like we set this error in the `onBlur` for all the other Radius components, but did it here for MGRS in the `onChange`. I don't think we had any reason to do it this way IIRC, @hayleynorton ?

**Review by Lambeaux — APPROVED** [review]

**Review by andrewzimmer — APPROVED** [review]

File: ui/packages/catalog-ui-search/src/main/webapp/react-component/location/point-radius.js (hunk 1 above)
Comment (hayleynorton) [reply ui/packages/catalog-ui-search/src/main/webapp/react-component/location/point-radius.js:242]: Not that I recall. Could just be we switched some things around but missed this one

**Review by cassandrabailey293 — APPROVED** [review]

**Comment by andrewzimmer** [comment]
Hero Successful 🎉 
- verified no errors entering values into point radius buffer

**Comment by andrewzimmer** [comment]
build now

**Comment by abel-connexta** [comment]
build now

**Comment by shaundmorris** [comment]
Merging. Build failed due to known docker issue.

**Pull request merged**